components/centos/centos-openshift-setup/Vagrantfile

```diff
@@ -13,6 +13,12 @@ IMAGE_TAG="v1.2.0"
 # You will use this IP address to connect to OpenShift web console.
 PUBLIC_ADDRESS="10.1.2.2"
 
+# Variables to enable HTTP/HTTPS proxy
+  
+#PROXY = "<proxy_server>:<port>"
```

Comment:
We need to clearly mention how we expect the user to put proxy server information i.e. http://<ip>:<port> Vs <ip> : <port>. May be we can add an dummy example.